Makefile

```diff
@@ -115,21 +115,21 @@ test: fmt vet envtest ## Run tests.
 
 .PHONY: build
 build: fmt vet bin/vm-builder ## Build all neonvm binaries.
-	go build -o bin/controller       neonvm/main.go
-	go build -o bin/vxlan-controller neonvm/tools/vxlan/controller/main.go
-	go build -o bin/runner           neonvm/runner/*.go
+	GOOS=linux go build -o bin/controller       neonvm/main.go
+	GOOS=linux go build -o bin/vxlan-controller neonvm/tools/vxlan/controller/main.go
+	GOOS=linux go build -o bin/runner           neonvm/runner/*.go
```

Comment:
What happens here if you're not on linux? does it just build binaries that can't be run locally?

Reply:
No, it just won't compile:

```
CGO_ENABLED=0 go vet ./...
# github.com/containerd/cgroups/v3
../../go/pkg/mod/github.com/containerd/cgroups/v3@v3.0.1/utils.go:63:13: undefined: unix.CGROUP2_SUPER_MAGIC
../../go/pkg/mod/github.com/containerd/cgroups/v3@v3.0.1/utils.go:70:23: undefined: unix.CGROUP2_SUPER_MAGIC
# github.com/containerd/cgroups/v3/cgroup2
../../go/pkg/mod/github.com/containerd/cgroups/v3@v3.0.1/cgroup2/devicefilter.go:103:24: undefined: unix.BPF_DEVCG_DEV_CHAR
../../go/pkg/mod/github.com/containerd/cgroups/v3@v3.0.1/cgroup2/devicefilter.go:105:24: undefined: unix.BPF_DEVCG_DEV_BLOCK
../../go/pkg/mod/github.com/containerd/cgroups/v3@v3.0.1/cgroup2/devicefilter.go:124:22: undefined: unix.BPF_DEVCG_ACC_READ
../../go/pkg/mod/github.com/containerd/cgroups/v3@v3.0.1/cgroup2/devicefilter.go:126:22: undefined: unix.BPF_DEVCG_ACC_WRITE
../../go/pkg/mod/github.com/containerd/cgroups/v3@v3.0.1/cgroup2/devicefilter.go:128:22: undefined: unix.BPF_DEVCG_ACC_MKNOD
../../go/pkg/mod/github.com/containerd/cgroups/v3@v3.0.1/cgroup2/devicefilter.go:134:34: undefined: unix.BPF_DEVCG_ACC_READ
../../go/pkg/mod/github.com/containerd/cgroups/v3@v3.0.1/cgroup2/devicefilter.go:134:60: undefined: unix.BPF_DEVCG_ACC_WRITE
../../go/pkg/mod/github.com/containerd/cgroups/v3@v3.0.1/cgroup2/devicefilter.go:134:87: undefined: unix.BPF_DEVCG_ACC_MKNOD
../../go/pkg/mod/github.com/containerd/cgroups/v3@v3.0.1/cgroup2/ebpf.go:51:17: undefined: unix.BPF_F_ALLOW_MULTI
../../go/pkg/mod/github.com/containerd/cgroups/v3@v3.0.1/cgroup2/manager.go:712:21: undefined: syscall.InotifyInit
../../go/pkg/mod/github.com/containerd/cgroups/v3@v3.0.1/cgroup2/manager.go:712:21: too many errors
# github.com/neondatabase/autoscaling/neonvm/tools/vxlan/controller
# [github.com/neondatabase/autoscaling/neonvm/tools/vxlan/controller]
vet: neonvm/tools/vxlan/controller/main.go:206:27: undefined: syscall.AF_BRIDGE
```